README.md

```diff
@@ -125,7 +133,19 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+genie = Genie.create(name: "Genie")
+Lamp.crate(wishes_remaining: 3)
+
+"Lamp belongs to Genie"
+Genie.create(name: "Genie")
+Lamp.crate(wishes_remaining: 3, genie_id: 1)
```

Comment:
Watch the details.  I know what you meant, but you used `crate`, instead of `create`, twice.